scripts/install_wgrib2.sh

```diff
@@ -32,10 +32,12 @@ cd grib2
 
 ## really someone needs to learn proper packaging conventions, but whatever
 CC=gcc FC=gfortran make
-ln -s /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
+cp /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
```

Comment:
Is there a reason why it is cp and not mv?

Reply:
No reason really.  I guess I sometimes do cp and then check before removing the source. But in this case mv would be just as good.